release/src/main.py

```diff
@@ -38,7 +35,18 @@
 COMMERCIAL_REPO="CommercialDistribution"
 bintray_target_repo="SonarQube-bintray"
 
-  
+
+content_type_json='application/json'
+
+class ReleaseRequest:
+  def __init__(self, org, project, buildnumber):
+    self.org = org
+    self.project = project
+    self.buildnumber = buildnumber
+
+  def is_sonarlint(self):
+    return 'sonarlint' in self.project
```

Comment:
Not sure, but is this 100% accurate ? What about `sonarlint-core` and `sonarlint-language-server` ? We want the same behaviour for them too ?

Reply:
Fair point; actually, a more accurate match would be `self.project.startswith('sonarlint')` - a.k.a match all projects whose name starts with `sonarlint` (`sonarlint-core, sonarlint-language-server, sonarlint-vscode, sonarlint-eclipse, sonarlint-intellij` - not sure about having to support `sonarlint-visual-studio`, but that would work too)